resources/conf.edn

```diff
@@ -0,0 +1,10 @@
+{:cassandra-session {:hosts #profile {:development ["localhost"]
+                                      :production ["node-0.cassandra.mesos"
+                                                   "node-1.cassandra.mesos"
+                                                   "node-2.cassandra.mesos"]}
```

Comment:
annoying we have to specify all the hosts (as there could potentially be a lot if things grew quite a bit)